server/tests/test_venvstack_sandbox.py

```
@@ -0,0 +1,372 @@
+    @pytest.mark.skipif(sys.platform != "darwin", reason="sandbox-exec is macOS only")
+    def test_sandbox_network_denied(self):
+        """Test that network access is denied in sandbox mode."""
+        code = """
+import urllib.request
+try:
+    urllib.request.urlopen("https://google.com", timeout=5)
+    network_allowed = True
+except Exception:
+    network_allowed = False
+"""
+        result, error = self.executor.execute(code, use_sandbox=True)
```

Ruff (0.14.13)

Unpacked variable `error` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)